project.clj

```diff
@@ -55,10 +55,14 @@
                  [org.postgresql/postgresql "9.2-1003-jdbc4"]
                  [clojureql "1.0.3"]
                  ;; MQ connectivity
-                 [clamq/clamq-activemq "0.4" :exclusions [org.slf4j/slf4j-api]]
-                 [org.apache.activemq/activemq-core "5.7.0" :exclusions [org.slf4j/slf4j-api org.fusesource.fuse-extra/fusemq-leveldb]]
+                 [org.apache.activemq/activemq-core "5.7.0"
+                  :exclusions [org.slf4j/slf4j-api
+                               org.fusesource.fuse-extra/fusemq-leveldb]]
+                 [org.apache.activemq/activemq-pool "5.7.0"
+                  :exclusions [org.slf4j/slf4j-api]]
+
                  ;; bridge to allow some spring/activemq stuff to log over slf4j
```

Comment:
@mullr ahh, so I thought you were talking about the exclusions, which I did double-check that we need.  With respect to the comment, I suppose it's accurate in the sense that we do still pull in some spring bits via activemq.